ansible_collections/arista/avd/roles/eos_designs/templates/underlay/logic.j2

```diff
@@ -14,6 +14,7 @@
                        "peer": peer,
                        "peer_interface": peer_uplink.interface,
                        "peer_type": peer_facts.switch.type,
+                       "peer_is_deployed": peer_facts.switch.is_deployed | arista.avd.default,
```

Comment:
```suggestion
                       "peer_is_deployed": peer_facts.switch.is_deployed,
```

the **is_deployed** is already set a default value (True) in the **eos_designs_facts.py**